projects/igniteui-angular/src/lib/core/styles/components/slider/_slider-component.scss

```diff
@@ -148,15 +101,54 @@
         @include e(ticks-label) {
             @extend %igx-slider__ticks-labels--disabled !optional;
         }
+    }
+}
+
+@each $t in $_igx-slider-thumbs {
+    @include b(igx-slider-thumb-#{$t}) {
+        @extend %igx-thumb-display !optional;
+
+        @include e(dot) {
+            @extend %igx-slider-thumb__dot !optional;
+        }
+
+        @include m(pressed) {
+            @include e(dot){
+                @extend %igx-slider-thumb__dot--pressed !optional;
+            }
+        }
+
+        @include m(active) {
+            @include e(dot){
+                @extend %igx-slider-thumb__dot--active !optional;
+            }
+        }
+
+        @include m(disabled) {
+            @extend %igx-thumb--disabled !optional;
+
+            @include e(dot){
+                @extend %igx-slider-thumb__dot--disabled !optional;
+            }
+        }
+    }
 
-        @each $t in $thumbs {
-            @include e(thumb-#{$t}) {
-                @extend %igx-thumb-display !optional;
-                @extend %igx-thumb--disabled !optional;
+    @include b(igx-slider-#{$t}) {
```

Comment:
The name of this block is a bit inconsistent. Rename it to `igx-slider-thumb-label-#{$t}` so it matches the block above. If you do not wish the child `label` element class to be generated as `igx-slider-thumb-label-to__label` you can change its name to `container` instead of `label`. In this way the child element selector will be `igx-slider-thumb-label-to__container`.